### PR TITLE
UniversalResults: update icon size to 18x18

### PR DESCRIPTION
**Pull request by oshi97**
"Current: Icons are 16px by 16px  on universal results
Expected: Update Icons to be 18px by 18px"

TEST=manual
check that in universal results title bar, the icon size is 18x18

### CODE REVIEW

File: src/ui/sass/modules/_Results.scss [hunk 1]
```diff
@@ -12,6 +12,8 @@ $results-title-bar-link-font-size: var(--yxt-font-size-md) !default;
 $results-title-bar-link-line-height: var(--yxt-line-height-xxlg) !default;
 $results-title-bar-link-font-weight: var(--yxt-font-weight-semibold) !default;
 
+$results-title-bar-icon-size: 18px !default;
```
Comment (creotutar) [inline src/ui/sass/modules/_Results.scss:15]: this was previously 1em. possible to keep that? or did we want specifically 18x18?

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss:15]: checked the mocks again and we want explicitly 18x18 (oversight by me)

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss:15]: actually hmm that doesn't necessarily mean we want 18px by 18px the mocks only operate in px

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss:15]: Does the aeb set the yxt-Results-titleIconWrapper font-size to 18px? probably best just to do whatever the aeb is doing

Comment (oshi97) [reply src/ui/sass/modules/_Results.scss:15]: (through hangouts) aeb has default font-size 18px from body.Text

File: src/ui/sass/modules/_Results.scss [hunk 2]
```diff
@@ -164,8 +168,6 @@ $results-cards-margin: calc(var(--yxt-base-spacing) / 2) !default;
   &-title svg, &-title img 
   {
     color: var(--yxt-color-brand-primary);
-    height: var(--yxt-base-spacing);
```
Comment (creotutar) [inline src/ui/sass/modules/_Results.scss:167]: Just a note: as we confirmed offline, this height and width default to 1em x 1em in the Icon.scss for `Icon svg` and `Icon-image`. Thus we do not need to override it here.

**Review by creotutar — APPROVED** [review]
🚀 🚀 🚀 LGTM

**Pull request merged**